### PR TITLE
A sample app in Rust/Actix

### PR DESCRIPTION
**Pull request by FabienD**
The first version of a Rust sample app. May be need improvements, more tests, open to discuss :-)

### CODE REVIEW

File: samples/start-app/rust-actix/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+env_logger = "0.10"
+tracing-actix-web = "0.7"
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+log = "0.4"
+tracing-log = "0.1"
+tracing-bunyan-formatter = "0.3"
+
+[dev-dependencies]
+reqwest = "0.11"
```
Comment (docteurklein) [inline samples/start-app/rust-actix/Cargo.toml:44]: both in dev-deps and deps?

File: samples/start-app/rust-actix/src/configuration.rs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+static PIM_AUTHORIZATION_SCOPES: &str =
+    "openid email profile read_channel_localization read_channel_settings";
+
+impl Settings {
+    pub fn get(path: Option<String>) -> Self {
+        self::Settings::load_env_var(path);
+
+        let app_host = env::var("APP_HOST").expect("APP_HOST is not set in .env file");
+        let app_port = env::var("APP_PORT").expect("APP_PORT is not set in .env file");
+        let session_key = env::var("SESSION_KEY").expect("SESSION_KEY is not set in .env file");
```
Comment (docteurklein) [inline samples/start-app/rust-actix/src/configuration.rs:27]: technically it misses an env var, doesn't have to come from .env file.

**Review by docteurklein — COMMENTED** [review]
noice

File: samples/start-app/rust-actix/Cargo.toml (hunk 1 above)
Comment (FabienD) [reply samples/start-app/rust-actix/Cargo.toml]: Oh, yes, it's removed on "dev-dependencies".

File: samples/start-app/rust-actix/src/configuration.rs (hunk 2 above)
Comment (FabienD) [reply samples/start-app/rust-actix/src/configuration.rs]: True, messages are updated.

**Review by tomglvng — APPROVED** [review]
Very nice !

**Pull request merged**